ansible_base/rbac/models.py

```diff
@@ -430,7 +438,9 @@ class RoleTeamAssignment(AssignmentBase):
         help_text=_("The role definition which defines permissions conveyed by this assignment"),
         related_name='team_assignments',
     )
-    team = models.ForeignKey(settings.ANSIBLE_BASE_TEAM_MODEL, on_delete=models.CASCADE, related_name='role_assignments')
+    team = models.ForeignKey(
+        settings.ANSIBLE_BASE_TEAM_MODEL, on_delete=models.CASCADE, related_name='role_assignments', help_text=_("The team this role will be applied to.")
```

Comment:
Careful here, because a team can be both an actor and an object (can get permission, or have permission to it given to another actor).

This field is a team-as-actor field.

So this is the team that receives the permissions of the related role definition, to the related object. You can cut that down to

> The team that receives permissions